### ISSUE TITLE
Normalisation issues

### ISSUE THREAD

**Issue by vivoutlaw** [kenshohara/3D-ResNets-PyTorch#97]
Hi, 

I assume that for training the mean ([mean.py](https://github.com/kenshohara/3D-ResNets-PyTorch/blob/master/mean.py)) and std used for kinetics  were [110.63666788, 103.16065604, 96.29023126 ] and [38.7568578, 37.88248729, 40.02898126]. 

For feature extraction, the mean [(mean.py)](https://github.com/kenshohara/video-classification-3d-cnn-pytorch/blob/master/mean.py) and std is set to  [114.7748, 107.7354, 99.4750] and [1, 1, 1] for Kinetics ([classify.py](https://github.com/kenshohara/video-classification-3d-cnn-pytorch/blob/master/classify.py#L14)). I suppose, wrong normalisation will degrade the performance of the extracted features. 

Can you please let me know,  what mean and std were used for the kinetics models provided here? [Link-to-pretrained-models](https://drive.google.com/drive/folders/1zvl89AgFAApbH0At-gMuZSeQB_LpNP-M). Thanks and looking forward to your reply.

best, Vivek

**Comment by vivoutlaw** [comment]
https://github.com/kenshohara/3D-ResNets-PyTorch/issues/12

Thanks
